platform-dom/src/main/java/com/softicar/platform/dom/elements/time/day/DomDayPopover.java

```diff
@@ -1,7 +1,7 @@
 package com.softicar.platform.dom.elements.time.day;
 
 import com.softicar.platform.common.date.Day;
-import com.softicar.platform.dom.elements.popup.modal.DomPopover;
+import com.softicar.platform.dom.elements.popup.popover.action.DomPopover;
```

Comment:
why is `DomPopover` in an `action` subpackage?

Reply:
oh, that should not have been this way. removed the `action` package.